requirements/python2.txt

```diff
@@ -0,0 +1,3 @@
+# celery stops supporting python2 at 4.3.0
+# thus for python2 tests we can't use latest master anymore
+celery<4.3.0
```

Comment:
this will be less than 5.0, celery didn't stop python 2 at 4.3. the master will support py2 again and python 5 will be moved to celery 5 branch

Reply:
The dependency tree for 4.3.0 installs the alpha of 5.0 of vine, which then causes the tests to bomb with:

```
    class Thenable(Callable, metaclass=abc.ABCMeta):  # pragma: no cover
                                     ^
SyntaxError: invalid syntax
```

Reply:
oooh, wait, I think I see what you're saying.  Okay, one sec.

Reply:
you can pin vine

Reply:
I've updated it to use celery<5.0.0